samples/tlssurvey/src/main/kotlin/okhttp3/survey/ssllabs/UserAgentCapabilities.kt

```diff
@@ -4,7 +4,7 @@ import com.squareup.moshi.JsonClass
 
 
 @JsonClass(generateAdapter = true)
-data class UserAgentCapabilities(
+class UserAgentCapabilities(
```

Comment:
As a data class it ends up generating a class that calls `Integer.hashCode(int)`

Reply:
That's desugared to API 1 since like AGP 3.0, but probably easier to just avoid if it's not in public API